src/python/pants/backend/codegen/protobuf/scala/BUILD

```diff
@@ -1,7 +1,8 @@
 # Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
-python_sources(dependencies=[":lockfile"])
+python_sources(dependencies=[":lockfile", ":scala_sources"])
 resource(name="lockfile", source="scalapbc.default.lockfile.txt")
+resources(name="scala_sources", sources=["*.scala"])
```

Comment:
Nit: These probably don't need to be seperate targets.

Reply:
Done.